### ISSUE TITLE
[BUG]  New player on the Team bug  

### ISSUE THREAD

**Issue by zigmond** [akasolace/HO#941]
4.0 r2736

When I buy a new player, when I click on him on the squad tab I get
    java.lang.NullPointerException

When I try to put him in the lineup tab
It won't even me close the error box and I have to end the process from task manager
The player will stop causing am error only after his first match with the team (league/cup/Friendly) not tournament or ladder
java.lang.NullPointerException
	at module.playerOverview.SpielerDetailPanel.setLabels(SpielerDetailPanel.java:368)
	at module.playerOverview.SpielerDetailPanel.setSpieler(SpielerDetailPanel.java:275)
	at module.playerOverview.SpielerUebersichtsPanel.setPlayer(SpielerUebersichtsPanel.java:56)
	at core.gui.HOMainFrame.setActualSpieler(HOMainFrame.java:292)
	at module.lineup.LineupPanel$1.synchronizeSelection(LineupPanel.java:265)
	at module.lineup.LineupPanel$1.valueChanged(LineupPanel.java:242)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown Source)
	at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
	at module.lineup.LineupPanel$1.synchronizeSelection(LineupPanel.java:261)
	at module.lineup.LineupPanel$1.valueChanged(LineupPanel.java:244)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown Source)
	at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
	at module.playerOverview.SpielerUebersichtNamenTable.setSpieler(SpielerUebersichtNamenTable.java:45)
	at module.lineup.LineupPanel.setPlayer(LineupPanel.java:63)
	at core.gui.HOMainFrame.setActualSpieler(HOMainFrame.java:291)
	at module.playerOverview.SpielerUebersichtsPanel.lambda$addTableSelectionListeners$2(SpielerUebersichtsPanel.java:261)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown Source)
	at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
	at module.playerOverview.SpielerUebersichtsPanel.selectRow(SpielerUebersichtsPanel.java:236)
	at module.playerOverview.SpielerUebersichtsPanel.lambda$addTableSelectionListeners$1(SpielerUebersichtsPanel.java:249)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
	at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown Source)
	at javax.swing.JTable.changeSelectionModel(Unknown Source)
	at javax.swing.JTable.changeSelection(Unknown Source)
	at javax.swing.plaf.basic.BasicTableUI$Handler.adjustSelection(Unknown Source)
	at javax.swing.plaf.basic.BasicTableUI$Handler.mousePressed(Unknown Source)
	at java.awt.AWTEventMulticaster.mousePressed(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)

Windows 10

**Comment by akasolace** [comment]
@zigmond  thank you for reporting this bug. I will try to reproduce.

**Comment by zigmond** [comment]
Diolch :)
If there is anything else I can do, let me know please :)

Zig

**Comment by akasolace** [comment]
if you have a database where the bug you described is still present, you can share it ....

**Comment by zigmond** [comment]
Please remind me where .

**Comment by zigmond** [comment]
Is it the latest HRP ?

**Comment by zigmond** [comment]
HRF I mean

**Comment by akasolace** [comment]
no it would be a database folder (to be shared for example via https://wetransfer.com/ or similar service)

**Comment by akasolace** [comment]
next time you buy a player, if that causes the same issue you can share your db before it get fixed by the player playing a game.

**Comment by zigmond** [comment]
Using that link, hopefully being sent to you now

*Samuel Ockenhouse
<https://stage.hattrick.org/Club/Players/Player.aspx?playerId=457077732&BrowseIds=391207719,412734944,416870217,425468052,433729785,442333238,456081439,388476785,411345454,413745038,414804531,416334710,416922135,420250867,427600947,428839776,434671929,434401090,455970527,457003806,458285446,458491398,456739974,458060679,458403033,456177642,458690802,456838725,457091998,458587301,434764652,455404108,457077732,458131836,396808742,402676921,404972526,406808014,414838120,397616072>
*is the player in question at the moment so you will hopefully be able to
see it when clicking on him :)

**Comment by zigmond** [comment]
It says it was sent, please let me know if you got it/anything else I can do

**Comment by akasolace** [comment]
>It says it was sent, please let me know if you got it/anything else I can do

sent to me how ?   I guess you need to copy / paste the link in this chat ...

**Comment by zigmond** [comment]
Files sent to
notifications@github.com
1 item, 313 MB in total ・ Expires on 8 March, 2021
Thanks for using WeTransfer. We'll email you a confirmation as soon as your
files have been downloaded.

Recipients
notifications@github.com
Download link
https://we.tl/t-YvnbFY5JsL
1 item

db
Folder・7 items

Message
[BUG] New player on the Team bug (#941)

**Comment by zigmond** [comment]
Hope that's the one... :)

**Comment by akasolace** [comment]
@zigmond I tried with latest version and I can't reproduce ( can you please give it a try with https://github.com/akasolace/HO/releases/tag/dev) ....

**Comment by zigmond** [comment]
Which of the assets ? :)

**Comment by akasolace** [comment]
if you are in windows 10 and I imagine 64bit version, I suggest you to test using:  
HO-5.0.3436.0-portable-win-DEV-JRE.zip

**Comment by zigmond** [comment]
Can't
get an error of couldn't load main class :(

**Comment by akasolace** [comment]
This is strange, I just try it:  download it, unzip enter the folder and launch HO.exe, works just fine for me ....

**Comment by zigmond** [comment]
Yes, it just stuck on load latest data for about 15 minutes until I am fed
up and "killing" it from task manager :(

**Comment by akasolace** [comment]
so you mean that: you can launch HO! but you can't do download new data, right ? 
Can you please be super precise otherwise it is difficult to figure out what is going on ....

**Comment by zigmond** [comment]
Yes, sorry, at first it said couldn't load main class, but that, I guess
was due to permissions that needed to be ticked :)

Then came the load latest dat, gave it 15 minutes and at the end it on task
manager ...

**Comment by akasolace** [comment]
can you please tell me exactly what you are doing cause I can't reproduce.
I am doing the following:
   1. download HO with JRE portable version
   2. launch it (all good ) and close it 
   3. copy your db folder 
   4. launch HO!   (it launches ok ... conversion of the databasae takes 2-3 minutes because you skipped some version and there is a lot of conversion happening) 

![image](https://user-images.githubusercontent.com/1136496/109842943-dd783980-7c4a-11eb-969d-89db2967310c.png)

![image](https://user-images.githubusercontent.com/1136496/109842998-eb2dbf00-7c4a-11eb-9095-728b613192e6.png)

There is one thing however which is suprising and that I dont like is the size increase of your db during the upgrade 
which almost quadrupled:

![image](https://user-images.githubusercontent.com/1136496/109843320-3647d200-7c4b-11eb-855d-41cdf2d0eb7f.png)

**Comment by zigmond** [comment]
java.lang.NullPointerException: Cannot invoke
"core.training.TrainingWeekManager.getTrainingList()" because
"this.recentTrainings" is null
at core.training.TrainingManager.getRecentTrainings(TrainingManager.java:71)
at core.model.HOModel.storeTrainingsSinceLastDownload(HOModel.java:629)
at core.net.DownloadDialog.startDownload(DownloadDialog.java:337)
at core.net.DownloadDialog.actionPerformed(DownloadDialog.java:96)
at
java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
at
java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
at
java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
at
java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
at
java.desktop/javax.swing.plaf.basic.BasicButtonListener$Actions.actionPerformed(BasicButtonListener.java:330)
at
java.desktop/javax.swing.SwingUtilities.notifyAction(SwingUtilities.java:1810)
at
java.desktop/javax.swing.JComponent.processKeyBinding(JComponent.java:2900)
at
java.desktop/javax.swing.JComponent.processKeyBindings(JComponent.java:2948)
at java.desktop/javax.swing.JComponent.processKeyEvent(JComponent.java:2862)
at java.desktop/java.awt.Component.processEvent(Component.java:6391)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1950)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:870)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1139)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1009)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:835)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4871)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2769)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
at
java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
at
java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
at
java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
at
java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
at
java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
at java.desktop/java.awt.Dialog.show(Dialog.java:1070)
at java.desktop/java.awt.Component.show(Component.java:1717)
at java.desktop/java.awt.Component.setVisible(Component.java:1664)
at java.desktop/java.awt.Window.setVisible(Window.java:1028)
at java.desktop/java.awt.Dialog.setVisible(Dialog.java:1005)
at core.net.DownloadDialog.initComponents(DownloadDialog.java:250)
at core.net.DownloadDialog.<init>(DownloadDialog.java:84)
at core.gui.HOMainFrame.actionPerformed(HOMainFrame.java:325)
at
java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
at
java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
at
java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
at
java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:349)
at
java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Actions.actionPerformed(BasicMenuItemUI.java:967)
at
java.desktop/javax.swing.SwingUtilities.notifyAction(SwingUtilities.java:1810)
at
java.desktop/javax.swing.JComponent.processKeyBinding(JComponent.java:2900)
at
java.desktop/javax.swing.JMenuBar.processBindingForKeyStrokeRecursive(JMenuBar.java:699)
at
java.desktop/javax.swing.JMenuBar.processBindingForKeyStrokeRecursive(JMenuBar.java:706)
at
java.desktop/javax.swing.JMenuBar.processBindingForKeyStrokeRecursive(JMenuBar.java:706)
at java.desktop/javax.swing.JMenuBar.processKeyBinding(JMenuBar.java:677)
at
java.desktop/javax.swing.KeyboardManager.fireBinding(KeyboardManager.java:311)
at
java.desktop/javax.swing.KeyboardManager.fireKeyboardAction(KeyboardManager.java:297)
at
java.desktop/javax.swing.JComponent.processKeyBindingsForAllComponents(JComponent.java:2993)
at
java.desktop/javax.swing.JComponent.processKeyBindings(JComponent.java:2985)
at java.desktop/javax.swing.JComponent.processKeyEvent(JComponent.java:2862)
at java.desktop/java.awt.Component.processEvent(Component.java:6391)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1950)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:870)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1139)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1009)
at
java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:835)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4871)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2769)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
at
java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
at
java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

**Comment by zigmond** [comment]
Update

Managed to get it working, had to put d/b from 3.0 as 4.0 crashed it, but,
it does work, and the crashing due to player that hasn't played seemed to
be fixed. hoever
clicking on the training tab get an error
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at
java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at
java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at
java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at
core.training.FutureTrainingManager.previewPlayer(FutureTrainingManager.java:81)
at core.training.FutureTrainingManager.<init>(FutureTrainingManager.java:41)
at
module.training.ui.model.OutputTableModel.getValueAt(OutputTableModel.java:130)
at
java.desktop/javax.swing.table.TableRowSorter$TableRowSorterModelWrapper.getValueAt(TableRowSorter.java:269)
at
java.desktop/javax.swing.DefaultRowSorter.compare(DefaultRowSorter.java:966)
at
java.desktop/javax.swing.DefaultRowSorter$Row.compareTo(DefaultRowSorter.java:1391)
at
java.desktop/javax.swing.DefaultRowSorter$Row.compareTo(DefaultRowSorter.java:1381)
at
java.base/java.util.ComparableTimSort.countRunAndMakeAscending(ComparableTimSort.java:320)
at java.base/java.util.ComparableTimSort.sort(ComparableTimSort.java:202)
at java.base/java.util.Arrays.sort(Arrays.java:1040)
at java.desktop/javax.swing.DefaultRowSorter.sort(DefaultRowSorter.java:616)
at
java.desktop/javax.swing.DefaultRowSorter.allRowsChanged(DefaultRowSorter.java:863)
at java.desktop/javax.swing.JTable.notifySorter(JTable.java:4263)
at java.desktop/javax.swing.JTable.sortedTableChanged(JTable.java:4123)
at java.desktop/javax.swing.JTable.tableChanged(JTable.java:4400)
at
java.desktop/javax.swing.table.AbstractTableModel.fireTableChanged(AbstractTableModel.java:297)
at
java.desktop/javax.swing.table.AbstractTableModel.fireTableDataChanged(AbstractTableModel.java:199)
at
module.training.ui.model.OutputTableModel.fillWithData(OutputTableModel.java:158)
at module.training.ui.OutputPanel.update(OutputPanel.java:80)
at module.training.ui.OutputPanel.initialize(OutputPanel.java:73)
at
core.gui.comp.panel.LazyImagePanel.callInitialize(LazyImagePanel.java:145)
at
core.gui.comp.panel.LazyImagePanel$2.hierarchyChanged(LazyImagePanel.java:132)
at
java.desktop/java.awt.Component.processHierarchyEvent(Component.java:6781)
at java.desktop/java.awt.Component.processEvent(Component.java:6400)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.Component.createHierarchyEvents(Component.java:5628)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1470)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1466)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1466)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1466)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1466)
at java.desktop/java.awt.Component.show(Component.java:1680)
at java.desktop/java.awt.Component.show(Component.java:1717)
at java.desktop/java.awt.Component.setVisible(Component.java:1664)
at java.desktop/javax.swing.JComponent.setVisible(JComponent.java:2675)
at
java.desktop/javax.swing.JTabbedPane.fireStateChanged(JTabbedPane.java:420)
at
java.desktop/javax.swing.JTabbedPane$ModelListener.stateChanged(JTabbedPane.java:293)
at
java.desktop/javax.swing.DefaultSingleSelectionModel.fireStateChanged(DefaultSingleSelectionModel.java:143)
at
java.desktop/javax.swing.DefaultSingleSelectionModel.setSelectedIndex(DefaultSingleSelectionModel.java:74)
at
java.desktop/javax.swing.JTabbedPane.setSelectedIndexImpl(JTabbedPane.java:646)
at
java.desktop/javax.swing.JTabbedPane.setSelectedIndex(JTabbedPane.java:621)
at
java.desktop/javax.swing.plaf.basic.BasicTabbedPaneUI$Handler.mousePressed(BasicTabbedPaneUI.java:4092)
at
java.desktop/javax.swing.plaf.synth.SynthTabbedPaneUI$1.mousePressed(SynthTabbedPaneUI.java:298)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at java.desktop/java.awt.Component.processMouseEvent(Component.java:6611)
at
java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
at java.desktop/java.awt.Component.processEvent(Component.java:6379)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4919)
at
java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4545)
at
java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4489)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2769)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
at
java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
at
java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

**Comment by akasolace** [comment]
@zigmond  I just tried again with the database you share with me.
Using latest dev version. I copy your db and launch HO!
As said above first launch takes approx 2 minutes (lot of items conversion) but then it works well. Also clicking on the training table is just fine:

![image](https://user-images.githubusercontent.com/1136496/110132263-471f5180-7dcb-11eb-86a6-426b9051daf6.png)

I really can't reproduce what you are mentioning. It is really strange we do not experience the same behaviour.
The only "issue" I see is what I mentioned above regarding the inflation of the db size following the upgrade.

@tychobrailleur what do you think of this:   http://hsqldb.org/doc/2.0/guide/management-chapt.html#tspace_usage  ?

**Comment by zigmond** [comment]
Thank you!
Sadly, I can still confirm that the training tab brings this
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at
java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at
java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at
java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at
core.training.FutureTrainingManager.previewPlayer(FutureTrainingManager.java:81)
at core.training.FutureTrainingManager.<init>(FutureTrainingManager.java:41)
at
module.training.ui.model.OutputTableModel.getValueAt(OutputTableModel.java:130)
at
java.desktop/javax.swing.table.TableRowSorter$TableRowSorterModelWrapper.getValueAt(TableRowSorter.java:269)
at
java.desktop/javax.swing.DefaultRowSorter.compare(DefaultRowSorter.java:966)
at
java.desktop/javax.swing.DefaultRowSorter$Row.compareTo(DefaultRowSorter.java:1391)
at
java.desktop/javax.swing.DefaultRowSorter$Row.compareTo(DefaultRowSorter.java:1381)
at
java.base/java.util.ComparableTimSort.countRunAndMakeAscending(ComparableTimSort.java:320)
at java.base/java.util.ComparableTimSort.sort(ComparableTimSort.java:202)
at java.base/java.util.Arrays.sort(Arrays.java:1040)
at java.desktop/javax.swing.DefaultRowSorter.sort(DefaultRowSorter.java:616)
at
java.desktop/javax.swing.DefaultRowSorter.allRowsChanged(DefaultRowSorter.java:863)
at java.desktop/javax.swing.JTable.notifySorter(JTable.java:4263)
at java.desktop/javax.swing.JTable.sortedTableChanged(JTable.java:4123)
at java.desktop/javax.swing.JTable.tableChanged(JTable.java:4400)
at
java.desktop/javax.swing.table.AbstractTableModel.fireTableChanged(AbstractTableModel.java:297)
at
java.desktop/javax.swing.table.AbstractTableModel.fireTableDataChanged(AbstractTableModel.java:199)
at
module.training.ui.model.OutputTableModel.fillWithData(OutputTableModel.java:158)
at module.training.ui.OutputPanel.update(OutputPanel.java:80)
at module.training.ui.OutputPanel.initialize(OutputPanel.java:73)
at
core.gui.comp.panel.LazyImagePanel.callInitialize(LazyImagePanel.java:145)
at
core.gui.comp.panel.LazyImagePanel$2.hierarchyChanged(LazyImagePanel.java:132)
at
java.desktop/java.awt.Component.processHierarchyEvent(Component.java:6781)
at java.desktop/java.awt.Component.processEvent(Component.java:6400)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at java.desktop/java.awt.Component.addNotify(Component.java:7085)
at java.desktop/java.awt.Container.addNotify(Container.java:2789)
at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
at java.desktop/java.awt.Container.addNotify(Container.java:2800)
at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
at java.desktop/java.awt.Container.addNotify(Container.java:2800)
at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
at java.desktop/java.awt.Container.addNotify(Container.java:2800)
at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4783)
at java.desktop/java.awt.Container.addImpl(Container.java:1146)
at java.desktop/java.awt.Container.add(Container.java:997)
at
module.training.TrainingModulePanel.initComponents(TrainingModulePanel.java:82)
at
module.training.TrainingModulePanel.initialize(TrainingModulePanel.java:33)
at core.gui.comp.panel.LazyPanel.callInitialize(LazyPanel.java:150)
at core.gui.comp.panel.LazyPanel$2.hierarchyChanged(LazyPanel.java:137)
at
java.desktop/java.awt.Component.processHierarchyEvent(Component.java:6781)
at java.desktop/java.awt.Component.processEvent(Component.java:6400)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.Component.createHierarchyEvents(Component.java:5628)
at
java.desktop/java.awt.Container.createHierarchyEvents(Container.java:1470)
at java.desktop/java.awt.Component.show(Component.java:1680)
at java.desktop/java.awt.Component.show(Component.java:1717)
at java.desktop/java.awt.Component.setVisible(Component.java:1664)
at java.desktop/javax.swing.JComponent.setVisible(JComponent.java:2675)
at
java.desktop/javax.swing.JTabbedPane.fireStateChanged(JTabbedPane.java:420)
at
java.desktop/javax.swing.JTabbedPane$ModelListener.stateChanged(JTabbedPane.java:293)
at
java.desktop/javax.swing.DefaultSingleSelectionModel.fireStateChanged(DefaultSingleSelectionModel.java:143)
at
java.desktop/javax.swing.DefaultSingleSelectionModel.setSelectedIndex(DefaultSingleSelectionModel.java:74)
at
java.desktop/javax.swing.JTabbedPane.setSelectedIndexImpl(JTabbedPane.java:646)
at
java.desktop/javax.swing.JTabbedPane.setSelectedIndex(JTabbedPane.java:621)
at
java.desktop/javax.swing.plaf.basic.BasicTabbedPaneUI$Handler.mousePressed(BasicTabbedPaneUI.java:4092)
at
java.desktop/javax.swing.plaf.synth.SynthTabbedPaneUI$1.mousePressed(SynthTabbedPaneUI.java:298)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at
java.desktop/java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:287)
at java.desktop/java.awt.Component.processMouseEvent(Component.java:6611)
at
java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
at java.desktop/java.awt.Component.processEvent(Component.java:6379)
at java.desktop/java.awt.Container.processEvent(Container.java:2263)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4990)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at
java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4919)
at
java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4545)
at
java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4489)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2769)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4822)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
at
java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
at
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
at
java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
at
java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
at
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
at
java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

If there is anything I can do to provide help with that, please let me know
:)

**Comment by akasolace** [comment]
@zigmond I tried again and I can't reproduce. Can you share with me the database with which you have this error because I can't reproduce the bug with the database you shared with me on top of this message ....

**Comment by zigmond** [comment]
Sent. I hope

**Comment by zigmond** [comment]
Files sent to
notifications@github.com
1 item, 1.19 GB in total ・ Expires on 15 March, 2021
Thanks for using WeTransfer. We'll email you a confirmation as soon as your
files have been downloaded.

Recipients
notifications@github.com
Download link
https://we.tl/t-VtDkpfSb0Y
1 item

DB HO-5.0.3551.0-portable-win-DEV-JRE
Folder・6 items

Message
[BUG] New player on the Team bug (#941- Training Tab Error

**Comment by akasolace** [comment]
so I just try again with the db you just shared with me and again I can't reproduce:

![image](https://user-images.githubusercontent.com/1136496/110316882-a4094a80-800b-11eb-8c2d-85132357992e.png)


![image](https://user-images.githubusercontent.com/1136496/110316903-ad92b280-800b-11eb-8025-ff7d3a57c831.png)

**Comment by zigmond** [comment]
Probably a problem at my end :(
Thank you for trying :)

**Comment by akasolace** [comment]
yes this is strange ... don't see how using the same version and the same database can produce different results ...

**Comment by zigmond** [comment]
i just ffound another error now
java.lang.NullPointerException
at
module.playerOverview.SpielerDetailPanel.setLabels(SpielerDetailPanel.java:368)
at
module.playerOverview.SpielerDetailPanel.setSpieler(SpielerDetailPanel.java:275)
at
module.playerOverview.SpielerUebersichtsPanel.setPlayer(SpielerUebersichtsPanel.java:56)
at core.gui.HOMainFrame.setActualSpieler(HOMainFrame.java:292)
at
module.playerOverview.SpielerUebersichtsPanel.lambda$addTableSelectionListeners$2(SpielerUebersichtsPanel.java:261)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown
Source)
at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
at
module.playerOverview.SpielerUebersichtNamenTable.setSpieler(SpielerUebersichtNamenTable.java:45)
at
module.playerOverview.SpielerUebersichtsPanel.setPlayer(SpielerUebersichtsPanel.java:54)
at core.gui.HOMainFrame.setActualSpieler(HOMainFrame.java:292)
at module.lineup.LineupPanel$1.synchronizeSelection(LineupPanel.java:265)
at module.lineup.LineupPanel$1.valueChanged(LineupPanel.java:242)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown
Source)
at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
at module.lineup.LineupPanel$1.synchronizeSelection(LineupPanel.java:261)
at module.lineup.LineupPanel$1.valueChanged(LineupPanel.java:244)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.fireValueChanged(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.changeSelection(Unknown Source)
at javax.swing.DefaultListSelectionModel.setSelectionInterval(Unknown
Source)
at javax.swing.JTable.setRowSelectionInterval(Unknown Source)
at
module.playerOverview.SpielerUebersichtNamenTable.setSpieler(SpielerUebersichtNamenTable.java:45)
at module.lineup.LineupPanel.setPlayer(LineupPanel.java:63)
at core.gui.HOMainFrame.setActualSpieler(HOMainFrame.java:291)
at
module.lineup.PlayerPositionPanel.itemStateChanged(PlayerPositionPanel.java:274)
at javax.swing.JComboBox.fireItemStateChanged(Unknown Source)
at javax.swing.JComboBox.selectedItemChanged(Unknown Source)
at javax.swing.JComboBox.contentsChanged(Unknown Source)
at javax.swing.AbstractListModel.fireContentsChanged(Unknown Source)
at javax.swing.DefaultComboBoxModel.setSelectedItem(Unknown Source)
at javax.swing.JComboBox.setSelectedItem(Unknown Source)
at javax.swing.JComboBox.setSelectedIndex(Unknown Source)
at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(Unknown
Source)
at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
at java.awt.Component.processMouseEvent(Unknown Source)
at javax.swing.JComponent.processMouseEvent(Unknown Source)
at javax.swing.plaf.basic.BasicComboPopup$1.processMouseEvent(Unknown
Source)
at java.awt.Component.processEvent(Unknown Source)
at java.awt.Container.processEvent(Unknown Source)
at java.awt.Component.dispatchEventImpl(Unknown Source)
at java.awt.Container.dispatchEventImpl(Unknown Source)
at java.awt.Component.dispatchEvent(Unknown Source)
at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
at java.awt.Container.dispatchEventImpl(Unknown Source)
at java.awt.Window.dispatchEventImpl(Unknown Source)
at java.awt.Component.dispatchEvent(Unknown Source)
at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
at java.awt.EventQueue.access$500(Unknown Source)
at java.awt.EventQueue$3.run(Unknown Source)
at java.awt.EventQueue$3.run(Unknown Source)
at java.security.AccessController.doPrivileged(Native Method)
at
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown
Source)
at
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown
Source)
at java.awt.EventQueue$4.run(Unknown Source)
at java.awt.EventQueue$4.run(Unknown Source)
at java.security.AccessController.doPrivileged(Native Method)
at
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown
Source)
at java.awt.EventQueue.dispatchEvent(Unknown Source)
at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
at java.awt.EventDispatchThread.run(Unknown Source)

seem to affect only players that are part of the NT (I have two of them) O
Arna and M Woaks. Trying to put them in any slot in the line up will bring
this error and I have to crash HO from task manager.
This is not on 5.0 Dev, but only on 4.0 r2736
Same database I sent you.

**Comment by akasolace** [comment]
If this bug (crash when putting player of NT in lineup) also exist with latest dev version please open a new ticket (I am not sure if you meant it is not on DEV version or if you did not tested it).  I will close this one  as it is becoming too mixed up. 
Thank you for reporting those issues.

**Comment by zigmond** [comment]
I will open a new one in a minute, but, just to be clear. It crushes on 4,
but, it is working fine on Dev 5 :)

**Comment by akasolace** [comment]
ah ok so please don't open a ticket ... we are now only focusing on 5.0 release which hopefully should not come in too much time (at least a working beta)

**Comment by zigmond** [comment]
Sorry, too late, opened one, please close it :)
